apps/mark-scan/backend/src/electrical_testing/background.ts

```diff
@@ -1,9 +1,40 @@
-import { extractErrorMessage, Result, sleep } from '@votingworks/basics';
+import { join } from 'node:path';
+import { promises as fs } from 'node:fs';
+import PdfDocument from 'pdfkit';
+import { Buffer } from 'node:buffer';
 
+import {
+  err,
+  ok,
+  extractErrorMessage,
+  Result,
+  sleep,
+  assertDefined,
+} from '@votingworks/basics';
+import {
+  getPaperHandlerDriver,
+  isPaperInScanner,
+  isPaperJammed,
+  isPaperReadyToLoad,
+} from '@votingworks/custom-paper-handler';
+import { LogEventId } from '@votingworks/logging';
+import { DateTime } from 'luxon';
 import { constructAuthMachineState } from '../util/auth';
-import { ServerContext } from './context';
+import {
+  printBallotChunks,
+  scanAndSave,
+} from '../custom-paper-handler/application_driver';
+import { type ElectricalTestingServerContext } from './server';
 
 const CARD_READ_INTERVAL_SECONDS = 5;
+const PAPER_HANDLER_POLL_INTERVAL_MS = 250;
+const PAPER_LOAD_LOG_INTERVAL_MS = 5000;
+
+const TEST_DOC = {
+  DPI: 72,
```

Comment:
Tested 200 DPI and confirmed 72 is the only value that results in the intended 8x11"